contracts/CO30_Rob_Byelomorye.IslaPera/edit_me/briefing/objectives.html

```diff
@@ -0,0 +1,10 @@
+<font color='#E3D310'>PRIMARY</font>
+- Rescue all hostages and escort them back to FOB Roger de Flor
+- Defuse all explosives planted at Presa Alta
+
+<font color='#595305'>SECONDARY</font>
+- Locate and destroy the enemy motar position
+- Clear Station Mike-69 of all enemy forces
+
+<font color='#E3D310'>RESTRICTIONS</font>
+- Limit civilian casualties
```

Comment:
This is always implied and doesn't need to be included unless there's anything additional